src/Microsoft.DotNet.Build.Tasks/PackageFiles/sign.targets

```diff
@@ -34,7 +35,7 @@
   </PropertyGroup>
 
   <Target Name="OpenSourceSign"
-          Condition="'$(DelaySign)' == 'true' and '@(IntermediateAssembly)' != '' and '$(SkipSigning)' != 'true' and '$(UseOpenSourceSign)' == 'true'"
+          Condition="'$(DelaySign)' == 'true' and '@(IntermediateAssembly)' != '' and '$(SkipSigning)' != 'true' and '$(SignType)' == 'oss'"
```

Comment:
Can we keep the old argument as well as a condition? The reason is that the changes I made in coreclr depend on this target not getting executed depending on this specific property getting passed in.